ambry-api/src/main/java/com.github.ambry/config/ClusterMapConfig.java

```diff
@@ -25,11 +25,8 @@
   @Default("com.github.ambry.clustermap.FixedBackoffResourceStatePolicyFactory")
   public final String clusterMapResourceStatePolicyFactory;
 
-  /**
-   * The fixed timeout based resource state handling checks if we have had a 'threshold'
-   * number of consecutive errors, and if so, considers the resource as down for
-   * 'retry backoff' milliseconds.
-   */
+  // The fixed timeout based resource state handling checks if we have had a 'threshold' number of consecutive errors,
```

Comment:
Removed, as this is unnecessary as you say - the documentation is there in the implementation.